### PR TITLE
Full balance

### PR DESCRIPTION
**Pull request by Blazebrain**
Issue Number (if Applicable): Fixes #

# Description

Please include a summary of the changes and which issue is fixed / feature is added. 

# Pull Request - Checklist  

- [x] Initial Manual Tests Passed
- [x] Double check modified code and verify it with the feature/task requirements
- [x] Format code
- [x] Look for code duplication
- [x] Clear naming for variables and methods

### CODE REVIEW

File: cw_bitcoin/lib/electrum_balance.dart [hunk 1]
```diff
@@ -24,7 +24,11 @@ class ElectrumBalance extends Balance {
   final int frozen;
 
   @override
-  String get formattedAvailableBalance => bitcoinAmountToString(amount: confirmed - frozen);
+  String get formattedAvailableBalance {
+    bool isOutgoingTx = unconfirmed.isNegative;
+    final availableBalance = isOutgoingTx ? (confirmed + unconfirmed) : (confirmed);
+    return bitcoinAmountToString(amount: availableBalance);
```
Comment (OmarHatem28) [inline cw_bitcoin/lib/electrum_balance.dart:30]: should deduct `frozen`

File: lib/src/screens/dashboard/pages/balance_page.dart [hunk 2]
```diff
@@ -506,6 +515,49 @@ class BalanceRowWidget extends StatelessWidget {
                   ),
                 ],
               ),
+            if (hasAdditionalBalance)
```
Comment (OmarHatem28) [inline lib/src/screens/dashboard/pages/balance_page.dart:518]: wouldn't this also affect Monero wallet?

**Review by OmarHatem28 — CHANGES_REQUESTED** [review]

File: cw_bitcoin/lib/electrum_balance.dart (hunk 1 above)
Comment (Blazebrain) [reply cw_bitcoin/lib/electrum_balance.dart:31]: Yep, you're right. Added it in.

File: lib/src/screens/dashboard/pages/balance_page.dart (hunk 2 above)
Comment (Blazebrain) [reply lib/src/screens/dashboard/pages/balance_page.dart]: Yes it would, added another check to make sure this impacts only BTC and LTC

**Review by OmarHatem28 — APPROVED** [review]

**Pull request merged**